.github/workflows/lint.yml

```diff
@@ -5,11 +5,11 @@ on: push
 jobs:
   black:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
```

Comment:
The existence of v4 is worth mentioning to all of us in Zulip or at the next team meeting. Are there any noticeable differences?

Reply:
I didn't research any differences... just saw a deprecation notice w.r.t. v2 so I upgraded to the most recent version. It's probably worth mentioning to folks, even if I'm not very well informed about it aside from "it works!".